crates/bevy_ui/src/layout/mod.rs

```diff
@@ -292,10 +293,11 @@ with UI components as a child of an entity without UI components, your UI layout
         }
     });
 
-    for (camera_id, mut camera) in camera_layout_info.drain() {
+    for (_camera_id, mut camera) in camera_layout_info.drain() {
         let inverse_target_scale_factor = camera.scale_factor.recip();
 
-        ui_surface.compute_camera_layout(camera_id, camera.size, text_buffers, &mut font_system);
+        #[cfg(feature = "bevy_text")]
+        ui_surface.compute_camera_layout(_camera_id, camera.size, text_buffers, &mut font_system);
```

Comment:
If we don't call `compute_camera_layout` none of the nodes will be updated.

```suggestion
        ui_surface.compute_camera_layout(
            _camera_id,
            camera.size,
            #[cfg(feature = "bevy_text")]
            text_buffers,
            #[cfg(feature = "bevy_text")]
            &mut font_system,
        );
```